### DISCUSSION TITLE
Is it possible to get color in specified coordinates on tile layer?

### DISCUSSION THREAD

**Discussion by vkotov** [Mapsui/Mapsui#1707] [Q&A]
I want to determine if the current coordinate is above ground or above water. I have two colors tile layer. How I can get color of pixel on map at specified coordinates?

**Comment by pauldendulk** [comment]
At the moment there is nothing to support that scenario. 

A workaround may be to create a screenshot at the moment you click the map and used to screen coordinates of the MapInfo event to determine the pixel in screenshot image. But I think this is hard to do and I am not sure if this is actually possible.
